intern/service/boss/baseversion.py

```diff
@@ -15,6 +15,7 @@
 from abc import ABCMeta, abstractmethod
 from intern.resource.boss.resource import CoordinateFrameResource
 from requests import Request
+from requests import get
```

Comment:
This is no longer needed, right?

Reply:
Yeah was done in Clean Up commit haha- I guess you started your review before I committed that